apps/editors/templates/editors/base.html

```diff
@@ -38,24 +38,40 @@ <h1 class="site-title amo">
 {% block navbar %}
 <div id="navbar">
   <ul class="top">
-      <li class="top">
-        <a href="{{ url('editors.queue') }}" class="controller">
-          {{ _('Queues') }}</a>
+    <li class="top" id="listed-queues">
+      <a href="{{ url('editors.queue') }}" class="controller">
+        {{ _('Queues') }}</a>
+      <ul>
+        {% if queue_counts %}
+          <li><a href="{{ url('editors.queue_fast_track') }}">
+                {{ _('Fast Track') }} ({{ queue_counts['fast_track'] }})</a></li>
+          <li><a href="{{ url('editors.queue_nominated') }}">
+                {{ _('Full Reviews') }} ({{ queue_counts['nominated'] }})</a></li>
+          <li><a href="{{ url('editors.queue_pending') }}">
+                {{ _('Pending Updates') }} ({{ queue_counts['pending'] }})</a></li>
+          <li><a href="{{ url('editors.queue_prelim') }}">
+                {{ _('Preliminary Reviews') }} ({{ queue_counts['prelim'] }})</a></li>
+          <li><a href="{{ url('editors.queue_moderated') }}">
+                {{ _('Moderated Reviews') }} ({{ queue_counts['moderated'] }})</a></li>
+        {% endif %}
+      </ul>
+    </li>
+    {% if action_allowed('Addons', 'ReviewUnlisted') %}
+      <li class="top" id="unlisted-queues">
+        <a href="{{ url('editors.unlisted_queue') }}" class="controller">
+          {{ _('Unlisted Queues') }}</a>
         <ul>
-          {% if queue_counts %}
-            <li><a href="{{ url('editors.queue_fast_track') }}">
-                  {{ _('Fast Track') }} ({{ queue_counts['fast_track'] }})</a></li>
-            <li><a href="{{ url('editors.queue_nominated') }}">
-                  {{ _('Full Reviews') }} ({{ queue_counts['nominated'] }})</a></li>
-            <li><a href="{{ url('editors.queue_pending') }}">
-                  {{ _('Pending Updates') }} ({{ queue_counts['pending'] }})</a></li>
-            <li><a href="{{ url('editors.queue_prelim') }}">
-                  {{ _('Preliminary Reviews') }} ({{ queue_counts['prelim'] }})</a></li>
-            <li><a href="{{ url('editors.queue_moderated') }}">
-                  {{ _('Moderated Reviews') }} ({{ queue_counts['moderated'] }})</a></li>
+          {% if unlisted_queue_counts %}
```

Comment:
I've removed those... they were needed because the "abuse reports" page wasn't providing the proper context (with the queue counts). I've changed that, and thus removed those tests.